### ISSUE TITLE
Crash on 1.18.1

### ISSUE THREAD

**Issue by boasbakker** [gliscowo/interactic#5]
When adding interactic to my mod list minecraft always crashes, after removing it my minecraft doesn't crash anymore.
```
[23:41:45] [Render thread/WARN]: @Redirect conflict. Skipping interactic.mixins.json:MinecraftClientMixin->@Redirect::handleDropPower(Lnet/minecraft/class_746;Z)Z from mod interactic with priority 1000, already redirected by fvt.mixins.json:MinecraftClientMixin->@Redirect::onHandleDropSelectedItem(Lnet/minecraft/class_746;Z)Z from mod fvt with priority 1000
[23:41:45] [Render thread/WARN]: @Redirect conflict. Skipping mixins.tweakeroo.json:MixinMinecraftClient->@Redirect::onProcessRightClickBlock(Lnet/minecraft/class_636;Lnet/minecraft/class_746;Lnet/minecraft/class_638;Lnet/minecraft/class_1268;Lnet/minecraft/class_3965;)Lnet/minecraft/class_1269; from mod tweakeroo with priority 990, already redirected by fvt.mixins.json:MinecraftClientMixin->@Redirect::hijackInteractBlock(Lnet/minecraft/class_636;Lnet/minecraft/class_746;Lnet/minecraft/class_638;Lnet/minecraft/class_1268;Lnet/minecraft/class_3965;)Lnet/minecraft/class_1269; from mod fvt with priority 1000
[23:41:45] [Render thread/WARN]: Method overwrite conflict for getPausedTickDelta in hudtweaks.mixins.json:MinecraftClientAccessor, previously written by carpet.mixins.MinecraftClient_pausedShakeMixin. Skipping method.
[23:41:45] [Render thread/ERROR]: Minecraft has crashed!
net.fabricmc.loader.impl.FormattedException: java.lang.RuntimeException: Mixin transformation of net.minecraft.class_310 failed
	at Not Enough Crashes deobfuscated stack trace.(1.18.1+build.2) ~[?:?]
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:610) ~[fabric-loader-0.12.12.jar:?]
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:77) [fabric-loader-0.12.12.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23) [fabric-loader-0.12.12.jar:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?]
	at org.multimc.onesix.OneSixLauncher.launchWithMainClass(OneSixLauncher.java:210) [NewLaunch.jar:?]
	at org.multimc.onesix.OneSixLauncher.launch(OneSixLauncher.java:245) [NewLaunch.jar:?]
	at org.multimc.EntryPoint.listen(EntryPoint.java:143) [NewLaunch.jar:?]
	at org.multimc.EntryPoint.main(EntryPoint.java:34) [NewLaunch.jar:?]
Caused by: java.lang.RuntimeException: Mixin transformation of net.minecraft.class_310 failed
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.getPostMixinClassByteArray(KnotClassDelegate.java:252) ~[fabric-loader-0.12.12.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.tryLoadClass(KnotClassDelegate.java:150) ~[fabric-loader-0.12.12.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:155) ~[fabric-loader-0.12.12.jar:?]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:520) ~[?:?]
	at net.minecraft.client.main.Main.main(Main:208) ~[minecraft-1.18.1-client.jar:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?]
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:608) ~[fabric-loader-0.12.12.jar:?]
	... 10 more
Caused by: org.spongepowered.asm.mixin.transformer.throwables.MixinTransformerError: An unexpected critical error was encountered
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:392) ~[sponge-mixin-0.10.7+mixin.0.8.4.jar:0.10.7+mixin.0.8.4]
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClass(MixinTransformer.java:234) ~[sponge-mixin-0.10.7+mixin.0.8.4.jar:0.10.7+mixin.0.8.4]
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClassBytes(MixinTransformer.java:202) ~[sponge-mixin-0.10.7+mixin.0.8.4.jar:0.10.7+mixin.0.8.4]
	at io.gitlab.jfronny.libjf.unsafe.asm.AsmTransformer.transformClassBytes(AsmTransformer.java:77) ~[libjf-unsafe-v0-2.2.0-da4e9ff1952f7040.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.getPostMixinClassByteArray(KnotClassDelegate.java:247) ~[fabric-loader-0.12.12.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.tryLoadClass(KnotClassDelegate.java:150) ~[fabric-loader-0.12.12.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:155) ~[fabric-loader-0.12.12.jar:?]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:520) ~[?:?]
	at net.minecraft.client.main.Main.main(Main:208) ~[minecraft-1.18.1-client.jar:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?]
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:608) ~[fabric-loader-0.12.12.jar:?]
	... 10 more
Caused by: org.spongepowered.asm.mixin.injection.throwables.InjectionError: Critical injection failure: Redirector handleDropPower(Lnet/minecraft/class_746;Z)Z in interactic.mixins.json:MinecraftClientMixin from mod interactic failed injection check, (0/1) succeeded. Scanned 1 target(s). Using refmap interactic-refmap.json
	at org.spongepowered.asm.mixin.injection.struct.InjectionInfo.postInject(InjectionInfo.java:468) ~[sponge-mixin-0.10.7+mixin.0.8.4.jar:0.10.7+mixin.0.8.4]
	at org.spongepowered.asm.mixin.transformer.MixinTargetContext.applyInjections(MixinTargetContext.java:1381) ~[sponge-mixin-0.10.7+mixin.0.8.4.jar:0.10.7+mixin.0.8.4]
	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.applyInjections(MixinApplicatorStandard.java:1052) ~[sponge-mixin-0.10.7+mixin.0.8.4.jar:0.10.7+mixin.0.8.4]
	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.applyMixin(MixinApplicatorStandard.java:400) ~[sponge-mixin-0.10.7+mixin.0.8.4.jar:0.10.7+mixin.0.8.4]
	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.apply(MixinApplicatorStandard.java:325) ~[sponge-mixin-0.10.7+mixin.0.8.4.jar:0.10.7+mixin.0.8.4]
	at org.spongepowered.asm.mixin.transformer.TargetClassContext.apply(TargetClassContext.java:421) ~[sponge-mixin-0.10.7+mixin.0.8.4.jar:0.10.7+mixin.0.8.4]
	at org.spongepowered.asm.mixin.transformer.TargetClassContext.applyMixins(TargetClassContext.java:403) ~[sponge-mixin-0.10.7+mixin.0.8.4.jar:0.10.7+mixin.0.8.4]
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:363) ~[sponge-mixin-0.10.7+mixin.0.8.4.jar:0.10.7+mixin.0.8.4]
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClass(MixinTransformer.java:234) ~[sponge-mixin-0.10.7+mixin.0.8.4.jar:0.10.7+mixin.0.8.4]
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClassBytes(MixinTransformer.java:202) ~[sponge-mixin-0.10.7+mixin.0.8.4.jar:0.10.7+mixin.0.8.4]
	at io.gitlab.jfronny.libjf.unsafe.asm.AsmTransformer.transformClassBytes(AsmTransformer.java:77) ~[libjf-unsafe-v0-2.2.0-da4e9ff1952f7040.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.getPostMixinClassByteArray(KnotClassDelegate.java:247) ~[fabric-loader-0.12.12.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.tryLoadClass(KnotClassDelegate.java:150) ~[fabric-loader-0.12.12.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:155) ~[fabric-loader-0.12.12.jar:?]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:520) ~[?:?]
	at net.minecraft.client.main.Main.main(Main:208) ~[minecraft-1.18.1-client.jar:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?]
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:608) ~[fabric-loader-0.12.12.jar:?]
	... 10 more
```

Mods:
```
  [✔️] advancementinfo-1.18-fabric0.43.1-1.3
  [✔️] AFKPeace-1.18-4.6.5
  [✔️] alternate-current-mc1.18-1.1.0 (1)
  [✔️] antighost-1.18-fabric0.43.1-1.1.4
  [✔️] anvil-crush-1.18.1-1.6.0
  [✔️] anvil-crushing-recipes-1.3.2-b1+MC1.18-1.18.1
  [✔️] architectury-3.2.51
  [✔️] authme-mc1.18.1-2.2.0
  [✔️] autoconfig1u-3.3.1
  [✔️] beenfo-1.18-fabric0.43.1-1.3.3
  [✔️] BetterAdvancements-1.18.1-0.1.2.121
  [✔️] betterbeds-1.2.0
  [✔️] betterbiomeblend-1.17.0-1.1.4-fabric
  [✔️] BetterPingDisplay-Fabric-1.17.1-1.1
  [✔️] better-recipe-book-1.5.5-1.17.1
  [✔️] blur-2.4.1
  [✔️] bobby-3.1.0
  [✔️] brandpacket-0.0.2.1
  [✔️] brb-fabric-1.5.6
  [✔️] carpet-extra-1.18-1.4.56
  [✔️] Carpet-TCTC-Addition-fabric-mc1.18-Snapshot-1.1.6+d3d2466
  [✔️] carpet-tis-addition-mc1.18.1-v1.27.0
  [✔️] carpet-tis-addition-mc1.18.1-v1.27.3
  [✔️] chat-coords-0.1.2149b
  [✔️] Chunky-1.2.153
  [✔️] CITResewn-0.8.1-1.18
  [✔️] Clean F3 1.0.1 mc 1.17 & 1.18
  [✔️] clean-logs-1.0.1
  [✔️] ClearDespawn-1.18-rc4-1.1.5
  [✔️] clear-skies-fabric-mc118-2.0.88
  [✔️] clickthrough-1.18-fabric0.43.1-0.4
  [✔️] cloth-api-2.0.54
  [✔️] cloth-config-6.1.48-fabric
  [✔️] continuity-1.0.0+1.17
  [✔️] convenient-mobgriefing-2.0.2
  [✔️] CreativeOnePunch-1.3-universal
  [✔️] customsplashscreen-1.2.0
  [✔️] deathlog-0.2.2+1.18
  [✔️] dontdropit-2.3.6+1.18
  [✔️] DraggableResourcePacks-1.0.1
  [✔️] dual_riders-1.18.1-1.0.5
  [✔️] durabilityviewer-1.18-fabric0.43.1-1.10.2
  [✔️] dynamic-fps-2.0.6
  [✔️] dynamic-surroundings-1.01forge-andfabric
  [✔️] DynamicSurroundings-Fabric-1.17.1-0.0.4
  [✔️] easiercrafting-1.18-fabric0.43.1-1.7
  [✔️] easiervillagertrading-1.18-fabric0.43.1-1.5.4
  [✔️] EasyVanilla-0.3.3
  [✔️] eating-animation-1.2
  [✔️] EditSign-1.18-2.2.6
  [✔️] eggtab-2.1.0+1.16-fabric
  [✔️] enchlevel-langpatch-0.2.0
  [✔️] entity-outliner-1.2.0
  [✔️] environmentalcreepers-fabric-1.17.0-1.6.0
  [✔️] extrasounds-2.1.1+1.18
  [✔️] fabric-api-0.44.0+1.18
  [✔️] fabric-carpet-1.18-1.4.56+v211130
  [✔️] fabric-chunkpregen-0.3.4
  [✔️] fabric-language-kotlin-1.7.1+kotlin.1.6.10
  [✔️] fabricskyboxes-1.18-0.5.4
  [✔️] fabrishot-1.6.0
  [✔️] fallingleaves-1.8.1+1.18.1
  [✔️] ferritecore-4.0.0-fabric
  [✔️] flytre-lib-1.4.4
  [✔️] food_highlight-1.18.1-1.0.3
  [✔️] fvt-fabric-1.9.0-mc1.18.1
  [✔️] grabcraft-litematic-1.18-fabric0.43.1-0.5
  [✔️] grass-pass-1.18-1.1.0
  [✔️] here-be-no-dragons-1.0.0
  [✔️] HUDTweaks-1.2.1-beta
  [✔️] indium-1.0.2-alpha1+mc1.18
  [✔️] inspecio-1.1.0+1.17 (1)
  [✔️] interactic-0.1.7+1.18
  [✔️] InvMove-fabric-1.18-0.1.0
  [✔️] InvView-1.4.8-1.18+
  [✔️] iris-mc1.18.1-1.1.3
  [✔️] isometric-renders-0.2.7-1.18.1
  [✔️] itemscroller-fabric-1.18.0-0.15.0
  [✔️] joml-1.10.2
  [✔️] KeepHeadNames-1.4-1.18 (1)
  [✔️] keybind-search-1.17-1.0.0
  [✔️] krypton-0.1.6
  [✔️] Ksyxis-1.1
  [✔️] lambdabettergrass-1.2.1+1.17
  [✔️] lambdabettergrass-1.2.1+1.17 (1)
  [✔️] lambdynamiclights-2.1.0+1.17
  [✔️] lazydfu-0.1.2
  [✔️] light-overlay-6.0.0-fabric
  [✔️] litematica-fabric-1.18.0-0.0.0-dev.20211203.013300
  [✔️] lithium-fabric-mc1.18.1-0.7.6
  [✔️] Loading-Timer-1.5
  [✔️] malilib-fabric-1.18.0-0.10.0-dev.26
  [✔️] MasaGadget-fabric-1.18.1-2.0.6
  [✔️] mcwifipnp-1.5.0-1.18-fabric
  [✔️] mcwifipnp-1.5.3-fabric
  [✔️] MemoryUsageScreen-1.17-1.2
  [✔️] minihud-fabric-1.18.0-0.19.0-dev.20211204.230657
  [✔️] mixintrace-1.1.0+1.17
  [✔️] modmanager-1.2.1+1.18
  [✔️] modmenu-3.0.0
  [✔️] modmenu-3.0.1
  [✔️] MoreTags-1.4.8
  [✔️] multiconnect-1.5.6
  [✔️] multiconnect-1.5.7
  [✔️] name-lookup-1.0.2
  [✔️] nethermap-1.5.0-1.18
  [✔️] nmuk-1.0.1+mc1.18-pre1
  [✔️] no_angled_brackets-universal-1.1.0
  [✔️] no-nether-portal-overlay-1.0.2
  [✔️] notenoughcrashes-4.1.3+1.18-fabric
  [✔️] OfflineSkins-1.17.1-v2-fabric
  [✔️] oh-my-minecraft-client-21w41a-0.3.9-beta.df860a9
  [✔️] okzoomer-5.0.0-beta.3+1.17.1
  [✔️] Pehkui-3.0.0+1.14.4-1.18
  [✔️] PerfectAccuracy-1.2
  [✔️] PerfectAccuracy-1.2 (1)
  [✔️] PetOwner-1.8.2-1.18-pre4+
  [✔️] Pick-Block-Pro-1.5.4-beta-mc1.18
  [✔️] pinglist-1.3.2
  [✔️] plusls-carpet-addition-1.18-0.2.2
  [✔️] PresenceFootsteps-r34-1.17
  [✔️] QuickConnectButton-1.0
  [✔️] Raw-Ores-To-Blocks-0.1
  [✔️] reeses_sodium_options-mc1.18-1.2.3
  [✔️] replanter-fabric-1.17.1-1.3
  [✔️] seedmapper-b1.4.6
  [✔️] servertick-mc1.18-1.6
  [✔️] servertick-mc1.18-1.6 (1)
  [✔️] servux-fabric-1.18.0-0.1.0
  [✔️] silence-1.18-pre1-fabric0.42.2-1.0.1
  [✔️] slight-gui-modifications-2.0.2
  [✔️] Slyde-1.5.0
  [✔️] sodium-extra-mc1.18-0.3.7
  [✔️] sodium-fabric-mc1.18-0.4.0-alpha5+build.9
  [✔️] spark-fabric
  [✔️] starlight-1.0.0 fabric.d0a3220
  [✔️] statssearch-1.18-fabric0.43.1-1.0.1
  [✔️] stepup-1.18-fabric0.43.1-1.1
  [✔️] styledplayerlist-2.1.3+1.18
  [✔️] TieFix-1.7.1
  [✔️] timechanger-1.0.0
  [✔️] TP-Utils-1.5.11-beta-mc1.18
  [✔️] tweakeroo-fabric-1.18.0-0.10.0-dev.20211204.230726
  [✔️] ViaBackwards-4.1.1
  [✔️] viafabric-0.4.3+232-main
  [✔️] visualoverhaul-4.0.0
  [✔️] vmp-fabric-mc1.18.1-0.2.0+beta.2.0-all
  [✔️] voicechat-fabric-1.18.1-2.1.28
  [✔️] worldedit-mod-7.2.8
  [✔️] wthit-fabric-4.3.1
  [✔️] wthit-plugins-1.0.0
  [✔️] Xaeros_Minimap_21.22.5_Fabric_1.18
  [✔️] XaerosWorldMap_1.18.7_Fabric_1.18
```

**Comment by mtb-xt** [comment]
I'm having the same issue, my crashlog is here - https://paste.atlauncher.com/view/a6a50da4-22d2-4db8-bb63-b7e75d0291e5

**Comment by gliscowo** [comment]
Fixed by Flourick/FVT-fabric#89
